packages/google-tag-manager/package.json

```diff
@@ -7,5 +7,8 @@
   "publishConfig": {
     "access": "public"
   },
-  "main": "index.js"
+  "main": "index.js",
+  "dependencies": {
+    "lodash": "^4.17.15"
```

Comment:
Not a part of this PR but would you please change using lodash to [defu](https://github.com/jsless/defu)?